guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -94,8 +94,8 @@ Remote Execution result upload
 | 443 | TCP | HTTPS | Amazon EC2, Azure, Google GCE | Compute resources | Virtual machine interactions (query/create/destroy) (optional)
 ifdef::satellite[]
 ifeval::["{mode}" == "connected"]
-| 443 | TCP | HTTPS | cloud.redhat.com | Red{nbsp}Hat Cloud plugin API calls |
-| 443 | TCP | HTTPS | Red{nbsp}Hat Portal | SOS report | Assisting support cases (optional)
+| 443 | TCP | HTTPS | Red{nbsp}Hat Hybrid Cloud Console | Red{nbsp}Hat Cloud plugin API calls |
```

Comment:
If I'm writing firewall rules I prefer to have a service name. I can resolve `console.redhat.com` using DNS, I can't resolve a description.

Having said that: if it's on a CDN that may also be hard to figure out 100% reliably, but it at least gives a hint.

Reply:
Gotcha.